.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Please add the `.vs_code` folder to the gitignore as it contains some machine specific paths.